recipes/clrng/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "clRNG" %}
+{% set version = "1.0.0-beta" %}
+{% set sha256 = "12e46c89a577ea26ea8af2b11176b1c503c2b0a28a3766178f317f21f27324f4" %}
+
+{% set OSX_VARIANT = os.environ.get("OSX_VARIANT", "native") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version.replace("-", "").replace("beta", "b") }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/clMathLibraries/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - OSX_VARIANT
+
+{% if OSX_VARIANT != "native" %}
+  skip: True  # [not osx]
+{% else %}
+  #############################################################
+  # Skip Visual Studio 2015 32-bit builds due to a bug.       #
+  #                                                           #
+  # xref: https://github.com/clMathLibraries/clRNG/issues/11  #
+  #############################################################
+  skip: True  # [(win and py35) or (win32 and py36)]
```

Comment:
Skipping Python 3.5 as Python 3.5 and Python 3.6 use the same VC runtime version. So either will work for our purposes here. This is typical.

What is worth noting is that VS 2015 32-bit builds fail. This [log]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.14805/job/rlmkrvh1dr42rf20#L436 ) demonstrates the build failure. An issue has already been raised about this upstream. ( https://github.com/clMathLibraries/clRNG/issues/11 ) As there doesn't seem to be a patch we can use to fix this, we merely disable those builds. VS 2015 64-bit builds seem to build fine.